pkg/api/norman/customization/node/node.go

```diff
@@ -99,7 +102,24 @@ func (a ActionWrapper) ActionHandler(actionName string, action *types.Action, ap
 		return drainNode(actionName, apiContext, false)
 	case "stopDrain":
 		return drainNode(actionName, apiContext, true)
+	case "scaledown":
+		return scaledownNode(actionName, apiContext)
+	}
+	return nil
+}
+
+func scaledownNode(actionName string, apiContext *types.APIContext) error {
+	node, schema, err := getNodeAndSchema(apiContext)
+	if err != nil {
+		return err
 	}
+
+	values.PutValue(node, time.Now().Format(time.RFC3339), "scaledown")
```

Comment:
would need to change to scaledowntime now

Reply:
great catch, thanks for that